### ISSUE TITLE
websocket opened and then closed immediately.

### ISSUE THREAD

**Issue by vercha** [nathanboktae/mocha-phantomjs#249]
@nathanboktae 
I'm doing a IMSDK test with mocha-phantomjs, but confused with the issue as titled.
hope to find some advices.
the mocha test run finely with real chrome/safrai browser.
any advice will be appreciated.

mocha-phantomjs4.1.0
node 4.4.6;

![image](https://cloud.githubusercontent.com/assets/13897098/23888964/9480d658-08c5-11e7-9f07-3a1f5bed0d81.png)

**Comment by vercha** [comment]
I checked its node_modules, founds that inner phantomjs version is 1.9.7..can I replace it by the newest phantomjs2.1?

**Comment by nathanboktae** [comment]
Yeah you can use the `-p` option to specify your own phantomjs binary or
use [mocha-phantomjs-core](
https://github.com/nathanboktae/mocha-phantomjs-core)

**Comment by vercha** [comment]
@nathanboktae 
Thanks a lot!  I've tried using mocha-phantomjs-core, it helps.
and I wonder how to use  the `-p` option to specify my own phantomjs binary, 
I tried ： 
mocha-phantomjs -p ./node_modules/phantomjs/bin/phantomjs  test/test.html
it turns out to be wrong:
![image](https://cloud.githubusercontent.com/assets/13897098/23940025/66b63bda-099e-11e7-9a24-f185262a40d8.png)
is there any example ? 
and thanks again!
